src/com/itacademy/eshop/services/ShopService.java

```diff
@@ -23,7 +25,15 @@ private ArrayList<Product> createProducts() {
          * one per each category in Category enum.
          * Product should not have any reviews.
          */
-        throw new UnsupportedOperationException("createProducts() is not implemented yet");
+        ArrayList<Product> arrayList = new ArrayList<>();
```

Comment:
No type names as parameter names. Please use that that collection is holding inside. In this case `products`